.travis.yml

```diff
@@ -11,7 +11,9 @@ branches:
 
 install:
   - psql -c 'create database jarbas;' -U postgres
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install v6 && nvm use v6
```

Comment:
Do we really need this whole thing here? I mean, we should be able to get NodeJS 6 without all that hassle. Can you confirm you tried some of this strategies before getting there?

My first attempt would be to just add this to `.travis.yml`: 

```yaml
node_js:
  - "6"
```

This would be passed to `nvm` which should be available in the VM.

If it's not, we can try to install it and pass it manually with a shorter command `apt-get install nvm && nvm use 6`.

Does any of that work?